### ISSUE TITLE
"Is Inside Volume" node crashing crashing

### ISSUE THREAD

**Issue by jimruxton** [JacquesLucke/animation_nodes#1591]
I am finding that there are times when the "Is Inside Volume" node is freezing Blender. I am using Blender 2.90 on Ubuntu 20.04 installed via snap. Animation-Nodes is the latest version. Below is an image of my node tree.
![animationNode](https://user-images.githubusercontent.com/29495434/99871326-5d502b80-2ba7-11eb-9301-ca910fba0b72.png)
I am creating a number of Empties using a python script then looking to see if the empties are inside a volume. In the tree "Here" is a list of all the Empty positions. In this example I am just using a simple cube. There appear to be certain positions of the cube in the matrix of empties where everything freezes. If I hit ctrl-c in the console I can get control of Blender again but need to restart Blender to have Animation-Nodes work again. This is what the console shows after ctrl-c.
```

Traceback (most recent call last):
  File "/home/jim/.config/blender/2.90/scripts/addons/animation_nodes/execution/main_execution_unit.py", line 38, in executeUnit
    exec(self.executeCodeObject, self.executionData, self.executionData)
  File "execution: 'NodeTree'", line 43, in <module>
  File "/home/jim/.config/blender/2.90/scripts/addons/animation_nodes/utils/bvh.py", line 15, in isInsideVolume
    hits2 = countHits(bvhTree, vector, direction2)
  File "/home/jim/.config/blender/2.90/scripts/addons/animation_nodes/utils/bvh.py", line 29, in countHits
    location = bvhTree.ray_cast(location + offset, direction)[0]
KeyboardInterrupt
```
`` When I run the node outside of a loop the same result is happening. I am attaching 2 images. The first one showing a position where the program freezes and the second one where it outputs correct results although it seems fairly random when everything freezes as I move the cube around using a mouse. Any ideas what is happening here?
![cubeCrashes](https://user-images.githubusercontent.com/29495434/99871527-8bcf0600-2ba9-11eb-9fdf-927a23a56b12.png)
Freezes above:

![cubeNoCrash](https://user-images.githubusercontent.com/29495434/99871541-a4d7b700-2ba9-11eb-823c-ad5043da1c84.png)
No Freeze above:

**Comment by OmarEmaraDev** [comment]
Are you sure it is just not taking a very long time to execute? I can't replicate the issue on my end. Can you share the blend file?

**Comment by jimruxton** [comment]
Here is a blend file that crashes for me when you open it if animation nodes is running. When the tree is working and putting out locations inside the cube I am getting about a 4 msec frame time. Very strange how it sometimes freezes. I don't think it is just taking a long time to execute.
[insideVolumeCrashes.zip](https://github.com/JacquesLucke/animation_nodes/files/5578289/insideVolumeCrashes.zip)

**Comment by OmarEmaraDev** [comment]
Thanks! I can replicate the issue with this file.

**Comment by jimruxton** [comment]
Thanks for letting me know. Happy to do any testing to help resolve the
issue.

**Comment by OmarEmaraDev** [comment]
Fixed in 11f4ffb. Thanks for the report!
